From 03f32985c35cd2a4130e7b5bfe99b0e2b0b51956 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 15 Jul 2017 00:03:36 +0200
Subject: [PATCH] do not try to cross-run carla-lv2-export
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

---
 source/plugin/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/source/plugin/Makefile b/source/plugin/Makefile
index 1b4017ead..1ec221366 100644
--- a/source/plugin/Makefile
+++ b/source/plugin/Makefile
@@ -505,9 +505,7 @@ $(BINDIR)/carla.lv2/manifest.ttl: $(OBJDIR)/carla-lv2-export.cpp.o $(BINDIR)/car
 	@echo "Linking carla-lv2-export$(APP_EXT)"
 	$(SILENT)$(CXX) $< $(LIBS_START) $(LIBS) $(LIBS_END) $(NATIVE_LINK_FLAGS) -o $(BINDIR)/carla-lv2-export$(APP_EXT)
 
-	@echo "Generating LV2 ttl data"
-	$(SILENT)cd $(BINDIR) && $(EXE_WRAPPER) ./carla-lv2-export$(APP_EXT)
-	$(SILENT)cd $(BINDIR)/carla.lv2 && ln -sf ../*bridge-* ../carla-discovery-* .
+	@echo "LV2 ttl data creation is postponed"
 
 $(BINDIR)/carla-lv2-export$(APP_EXT): $(OBJDIR)/carla-lv2-export.cpp.o
 	-@mkdir -p $(BINDIR)
