From 1c871f1154cd67b00f1fc41d49fb94bddbf4b714 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 6 Dec 2019 18:11:09 +0100
Subject: [PATCH] Fix build for python3-only environments
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Thanks for upgrading waflib

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 waf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/waf b/waf
index 887215c..2538272 100755
--- a/waf
+++ b/waf
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Minimal waf script for projects that include waflib directly
 
